includes/functions.php

```diff
@@ -121,7 +121,16 @@ function wp_stream_is_vip() {
  * @return bool
  */
 function wp_stream_is_cron_enabled() {
-	return ( defined( 'DISABLE_WP_CRON' ) && DISABLE_WP_CRON ) ? false : true;
+	$is_cron_enabled = ( defined( 'DISABLE_WP_CRON' ) && DISABLE_WP_CRON ) ? false : true;
+
+	/**
+	 * Filter cron check
+	 *
+	 * @param bool $is_cron_enabled Whether cron is enabled or not.
+	 */
+	$is_cron_enabled = apply_filters( 'wp_stream_is_cron_enabled', $is_cron_enabled );
+
+	return $is_cron_enabled;
```

Comment:
@kucrut Could we cast this to boolean to ensure a predictable return value? Also, `$is_cron_enabled` isn't used anywhere apart from storing the filter value so maybe we can return the filtered value directly?

```php
return (bool) apply_filters( 'wp_stream_is_cron_enabled', $is_cron_enabled );
```